### PR TITLE
Create a single Default DID

### PR DESCRIPTION
**Pull request by osinakayah**

### CODE REVIEW

File: src/features/didManagement/didManagment-slice.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  await wallet.add({
+    ...didDocumentResolution,
+  });
+  await wallet.update({
+    ...keyDoc,
+    correlation: correlations,
+  });
+};
+
+const initializeDiDs = async () => {
```
Comment (mike-parkhill) [inline src/features/didManagement/didManagment-slice.js:32]: nitpick: inconsistent use of DID vs DiD

File: src/features/didManagement/didManagment-slice.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  if (keyDocs.length > 0) {
+    const didResolutionDocuments = await wallet.query({
+      type: 'DIDResolutionResponse',
+    });
+
+    if (didResolutionDocuments.length === 0) {
+      await createDID(keyDocs[0]);
+    }
+  } else {
+    const keyDoc = await createKeyDoc();
```
Comment (mike-parkhill) [inline src/features/didManagement/didManagment-slice.js:46]: Think I'm not understanding what's happening here. Are we creating a new keyDoc every time we load the wallet after the first time? Why? And why don't we create one the first time?

Comment (osinakayah) [reply src/features/didManagement/didManagment-slice.js:46]: No, we only create it if there is non.
Each time the wallet is loaded, we check if we have a keyDoc, if we don't we, create one, then we check if there is didDocument, if there is non, we create a didDocument.

Comment (mike-parkhill) [reply src/features/didManagement/didManagment-slice.js:46]: right, I see that now.

Thanks.

**Review by mike-parkhill — COMMENTED** [review]
Just linting issues left I think.

**Pull request merged**